lib/win32/screenshot/bitmap_maker.rb

```diff
@@ -40,31 +42,57 @@ class << self
                         [:long], :bool
         attach_function :release_dc, :ReleaseDC,
                         [:long, :long], :int
+        attach_function :bit_blt, :BitBlt,
+                        [:long, :int, :int, :int, :int, :long, :int, :int, :long], :bool
+
+        DIB_RGB_COLORS = 0
+        PW_RENDERFULLCONTENT = 0x00000002
+        SRCCOPY = 0x00CC0020
+
+        SM_XVIRTUALSCREEN = 76
+        SM_YVIRTUALSCREEN = 77
+        SM_CXVIRTUALSCREEN = 78
+        SM_CYVIRTUALSCREEN = 79
 
         def capture_all(hwnd, context)
           width, height = dimensions_for(hwnd, context)
-          capture_area(hwnd, context, 0, 0, width, height)
+          capture_area(hwnd, context, width: width, height: height)
```

Comment:
What's the point of this change of having `width: width` and `height: height`?

I have to admit that I have not written newer Ruby syntax and I'm not aware of all of its features, I would have expected that it is just passing a Hash, but looking at the method signature of `capture_area` it seems like keyword arguments? Maybe not add that extra complexity?